packages/epics/src/governance/components/document-vote-button.tsx

```diff
@@ -0,0 +1,42 @@
+import { Skeleton, Button } from '@hypha-platform/ui';
+
+interface DocumentVoteButtonProps {
+  voted?: boolean;
```

Comment:
Please one of:

```suggestion
  isVoted?: boolean;
```

or 
```suggestion
  hasVoted?: boolean;
```